distributed/tests/test_steal.py

```diff
@@ -1446,6 +1446,57 @@ def func(*args):
     assert (ntasks_per_worker < ideal * 1.5).all(), (ideal, ntasks_per_worker)
 
 
+def test_balance_steal_communication_heavy_tasks():
+    dependencies = {"a": 10, "b": 10}
```

Comment:
If I change this line to `{"a": 1e-6, "b": 1e-6}` the test remains green, and if go lower than that I instead get "ValueError: Expected a value larger than 16 integer but got 10."
 Are we sure we're actually testing anything here?

Reply:
This test breaks if we double the cost of the dependencies to `{"a": 20, "b": 20}` as the tasks have become _too_ expensive to move in this setup. By reducing the cost of the dependencies, we are checking whether we would move tasks that are cheap to move which is the desired behavior. Reducing the cost below `1e-6` creates trouble since we are calculating the actual size of the dependencies as the product of their specified cost and the available bandwidth.